package.json

```diff
@@ -65,7 +65,6 @@
   },
   "dependencies": {
     "@types/adlib": "^3.0.1",
-    "adlib": "^3.0.4",
```

Comment:
@dbouwman I assume this was a mistake to have `adlib` installed at the root. It's not used by any of the doc tooling or anything is it?

Reply:
I dont think so... likely a mistake